packages/jasmine/src/jasmine_runner.js

```diff
@@ -60,31 +63,35 @@ function main(args) {
   // second is always a flag to enable coverage or not
   const coverageArg = args[1];
   const enableCoverage = coverageArg === '--coverage';
+  // third arg is an optional seed
+  const seed = args[2] === '--seed' ? args[3] : null;
```

Comment:
this seems sad since any future flags need more special treatment. I wonder if we could just pass the residual args to jasmine's built-in command line parser?

Reply:
I know very little about that. Do you have any documentation you can point me at, or a recommended approach?

Reply:
I'm skipping this for now. I've added two optional arguments: `randomize` and `seed` along with documentation comments.